fhir-persistence/src/test/java/com/ibm/fhir/persistence/test/common/AbstractReverseChainTest.java

```diff
@@ -89,18 +90,18 @@ public void createResources() throws Exception {
         startTrx();
         // Organizations that will be referenced by a Patient
         savedOrg1 = org.toBuilder().active(com.ibm.fhir.model.type.Boolean.of(true)).build();
-        savedOrg1 = persistence.create(getDefaultPersistenceContext(), savedOrg1).getResource();
+        savedOrg1 = FHIRPersistenceTestSupport.create(persistence, getDefaultPersistenceContext(), savedOrg1).getResource();
```

Comment:
how would you feel about statically importing all these FHIRPersistenceTestSupport methods in all these abstract tests.  I think it might clean them up a bit visually.

Reply:
opened issue #3285 for this.